logistics/scripts/modules/packaging/nuget-helper.psm1

```diff
@@ -101,38 +101,20 @@ function Get-NuGetPackage {
     )
 
     $nuget = Install-NuGetCli $ToolsPath
```

Comment:
we should update the version in the script to download the appCommon package with your changes `1.4.1170`
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/c9eb7787b3863766f8a3732133ac5f5fad8d4b3c/Scripts/NuGet/prep-installer-package.ps1#L18